scripts/zones/North_Gustaberg_[S]/mobs/Olgoi-Khorkhoi.lua

```diff
@@ -13,6 +13,7 @@ end
 
 entity.onMobDespawn = function(mob)
     -- Sets to respawn between 90 to 120 minutes
+    UpdateNMSpawnPoint(mob:getID())
```

Comment:
Does this NM have rows in nm_spawn_points.sql ? if he does nobody ever commented the mobs name. if there are no rows this won't work as expected.

Reply:
lmao, he does in mob_spawn_points.sql. Is Olgoi even a NM? hmmm.

Reply:
incidentally needing a random spawn point is another sign he should be set up in the zone init - else every 1st spawn is the exact same spot

Reply:
I'll remove this on next commit. It's a worm, it's likely it won't go far anyways

Reply:
> I'll remove this on next commit. It's a worm, it's likely it won't go far anyways

that's not how we roll here. retail accurate or bust.